django-stubs/forms/boundfield.pyi

```diff
@@ -33,6 +34,9 @@ class BoundField:
     def __getitem__(self, idx: slice) -> list[BoundWidget]: ...
     @property
     def errors(self) -> ErrorList: ...
+    @property
+    def template_name(self) -> str: ...
+    def get_context(self) -> dict[str, type[Self]]: ...
```

Comment:
I think it should be `dict[str, Any]`: https://github.com/django/django/blob/e72049aa6302411d8cdf2e798e15fa38e76b92fc/django/forms/boundfield.py#L83

Reply:
Thank you for the good point :)